SiD/compact/SiD_o2_v03/HCalBarrel_Cu_v03.xml

```diff
@@ -0,0 +1,51 @@
+<lccdd>
+
+<!-----------------------------------------------------------------------------------------------
+    AHCal Barrel implementation of the layout from 70th SiD Optimization Meeting presentation 
+    from Ross Gordon McCoy (ross.mccoy@mavs.uta.edu) - June 30, 2016, updated Nov 20, 2016
+    Cu absorber version of o2_v03 included here for convenience - protopop@cern.ch - Dec 4, 2017
+    See https://www.evernote.com/l/AJ2HhEFs29NFZKbSV49f8xhNVTv-RsNqejc
```

Comment:
Could this be moved to confluence? I'm a bit concerned that this is linked to a personal account.

Reply:
Ross' email is in perhaps 3 of the files. I can remove it but I believe it will remain in the change history anyway. My email is listed in some of the files and I am not concerned.